RocketChat/Rocket.Chat
# feat: Update autotranslate on language preference change

## Proposed changes (including videos or screenshots)


Now every time the user updates their language to a different language and the `AutoTranslate_AutoEnableOnJoinRoom` is set, we need to update all of his subscriptions to reflect the new AutoTranslate language.

## Issue(s)


## Steps to test or reproduce


Follow [this](https://docs.rocket.chat/setup-and-configure/advanced-workspace-management/google-cloud) to enable autotranslate:

1. Join a room
2. Mark the autotranslate checkbox (inside the kebab menu)
3. Change the language setting to one that is different from the room
4. Go back to the room and check if all the next messages are translated to the users new language

## Further comments

TC-1076
